docs/rower_settings.md

```diff
@@ -1,76 +1,343 @@
+
+## Making sure the hardware is connected correctly and works as intended
+
+Before you physically connect anything to anything else, **check the electric properties of the rower** you are connecting to. Skipping this might destroy your Raspberry Pi as some rowers are known to exceed the Raspberry Pi electrical properties. For example, a Concept 2 RowErg provides 15V signals to the monitor, which will destroy the GPIO-ports. Other rowers provide signals aren't directly detectable by the raspberry Pi. For example, the Concept 2 Model C provides 0.2V pulses, thus staying below the detectable 1.8V treshold that the Raspberry Pi uses. Using a scope or a voltmeter is highly recommended. Please observe that the maximum input a Raspberry Pi GPIO pin can handle is 3.3V and 0.5A, and it will switch at 1.8V (see [this overview of the Raspberry Pi electrical properties](https://raspberrypi.stackexchange.com/questions/3209/what-are-the-min-max-voltage-current-values-the-gpio-pins-can-handle)). In our [GitHub Discussions](https://github.com/laberning/openrowingmonitor/discussions) there are some people who are brilliant with electrical connections, so don't be affraid to ask for help there. When you have a working solution, please report it so that we can include it in the documentation, allowing us to help others.
+
+Next, when the electric connection has been made, we need to look if the data is recieved well and has sufficient quality to be used. You can change `config/config.js` by
+
+  ```zsh
+  sudo nano /opt/openrowingmonitor/config/config.js
+  ```
+
+Here, you can change the setting for **createRawDataFiles** by setting:
+
+ ```js
+ createRawDataFiles: true,
+ ```
+
+You can use the following commands on the command line to restart after a config change (to activate new settings):
+
+  ```zsh
+  sudo systemctl restart openrowingmonitor
+  ```
+
+After rowing a bit, there should be a csv file created with raw data. Please read this data in Excel (it is in US format, so you might need to adapt it to your local settings), to check if it is sufficiently clean. After loading it into Excel, you can visualise it, and probably see something similar to the following:
+
+<img src="img/CurrentDt_curve.jpg" width="700">
+
+When the line goes up, the time between impulses from the flywheel goes up, and thus the flywheel is decellerating. When the line goes down, the time between impulses decreases, and thus the flywheel is accelerating. In the first decellerating flank, we see some noise, which Open Rowing Monitor an deal with perfectly. However, looking at the bottom of the first acceleration flank, we see a series of heavy downward spikes. This could be start-up noise, but it also could be systematic across the rowing session. This is problematic as it throws off both stroke detection and many metrics. Typically, it signals an issue in the mechenical construction of the sensor: the fram and sensor vibrate at high speeds, resulting in much noise.
```

Comment:
>Typically, it signals an issue in the mechenical construction of the sensor

I think this sentence is a slightly miss leading as for reed switches this is completely normal. Hall effect and optical sensors generally do not suffer from this as in their circuit some form of debounce filter is already included. So I think its not correct that we advise a person to change the sensor just because their signal has a noise that may be cleaned up (potentially in code).

>the fram and sensor vibrate at high speeds, resulting in much noise.

here (apart from the typo) I think its important to note that this is likely inaccurate. Hall sensors generally do not suffer from this issue, while even cheap reed switches can, generally withstand 20gram vibration for 11ms. For instance I use this type of reed:[datasheet-503548-pic-ms-213-3-cylindrical-reed-sensor-na-1-a-10-w.pdf](https://github.com/laberning/openrowingmonitor/files/9631789/datasheet-503548-pic-ms-213-3-cylindrical-reed-sensor-na-1-a-10-w.pdf)
Under environmental characteristics you can find the related metrics.

Finally:
> Better magnet aligning helps here, but sometimes it is inherent to cheaper sensors.

I think this statement is misleading as not price that really determines the amount of bounce. Bounce is a general issue. 

Sorry that I go on a lot about this bounce topic, but this is something that is not (or should not) be an issue for C2 as it uses a hall effect sensor. But it is probably something to consider/account for most of the other rowing machines (as reed is a simpler and easier choice, not to mention that hall effect sensor consumes battery power while reed does not).

Reply:
For V1Beta, this section of the text will be replaced by some additional text explaining how to calibrate the glitchFilter and configureClock.